vxquery-benchmark/src/main/resources/noaa-ghcn-daily/other_systems/mrql_scripts/run_mrql_tests.sh

```diff
@@ -40,7 +41,6 @@ if which programname >/dev/null;
 then
     echo "Sending out e-mail notification."
     SUBJECT="MRQL Tests Finished (${DATASET})"
-    EMAIL="ecarm002@ucr.edu"
```

Comment:
I'm confused here as well. This looks like it is actually sending you an email right?